src/schema/rules/datatypes/common_derivatives.yaml

```diff
@@ -0,0 +1,413 @@
+---
+anat_nonparametric_common:
+  $ref: schema.rules.datatypes.anat.nonparametric
```

Comment:
Part of the goal of #1012 was to eliminate the dependency on the folder structure and filenames for datatype rules. If the group names are unique, then I think we could have `schema.rules.datatypes.nonparametric`.

@effigies were the names unique?

Reply:
No, the names weren't unique, but we could make them unique if we reorganize. I think fully-qualified is fine as long as we update the paths when things get moved.